debian/pbspro-server.install

```diff
@@ -0,0 +1,308 @@
+etc/pbs_db_schema.sql /opt/pbs/etc
```

Comment:
@subhasisb, I'm seeing a huge amount of redundancy between this pull request and what is already present in the pbspro.spec file. If we have an opportunity to reduce/eliminate redundancy (which always results in increased maintenance cost), shouldn't we investigate it? All of the tools we need (rpmbuild) are available to generate an RPM for Debian/Ubuntu. It's just a matter of converting it once we have the RPM. There may be good reasons to utilize the native (.deb) package tools in this case, but they should outweigh the maintenance costs we will end up incurring. Whenever I see redundant code, it raises a red flag for me.